docs/DynamicESDL/DESDL-Mods/ESDLauth_feature.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<sect2 id="ESDL_Attributes_auth_feature">
+  <title><emphasis role="bold">auth_feature</emphasis></title>
+
+  <para>The auth_feature attribute (valid only for an ESPMethod) allows you to
+  specify means to authenticate user's permission to execute a method.</para>
+
+  <para><emphasis role="bold">Example:</emphasis></para>
+
+  <programlisting> ESPservice MyService
+
+ {
+   ESPmethod MyMethod1(MyMethod1Request, MyMethod1Response);
+   ESPmethod 
+    [
+     description("MyMethod Two"),
+     auth_feature("AllowMyMethod2"),
```

Comment:
@JamesDeFabia that should work. It would prob be very useful to mention as part of this process.